documentation/docs/is-cli/is_cli_examples.md

```diff
@@ -0,0 +1,103 @@
+# Examples
+
+In this example we will show how to create Identities, Channels and how to subscribe to a channel, authorize one Subscriber and write data into a Channel.
+
+
+### Create json files
+
+First create two files `identityA.json` and `identitiyB.json` for creating these Identities with the CLI in the next step. Rename for the second file the `username`.
+
+```json
+{
+	"username": "actorA",
+	"claimType": "User",
+	"claim": {
+		"name": "John",
+		"surname": "Smith"
+	}
+}
+```
+
+### Create the Identities
+```sh
+is create-identity -i examples/identityA.json -o identityA.json 
+```
+```sh
+is create-identity -i examples/identityB.json -o identityB.json 
+```
+
+### Create a Channel
+
+Next we will create a private Channel with a topic `logs` and the source `device` and save the created Channel in a file called `channelA.json`. For public Channels add the option `-pC`. For public Channels a subscriber is automatically authorized to the Channel but can't write into it. The user can read the history of this channel with the command `is read-channel-history`.
+
+```sh
+is create-channel -i identityA.json -t logs -s device -o channelA.json -pC test-channel
+```
+
+### Write to Channel
+
+The creator of the Channel can directly write into it.
+
+```sh
+is write-channel -i identityA.json -p "hello world" <Channel address of channelA>
```

Comment:
Change `write-channel` implementation and docs to receive channel address from a channel file (the channelA.json of the previous example). The command write-channel need to have a -c channelFile.json where read the channel address. In this way we avoid complex copy and paste